### DISCUSSION TITLE
Looking for maintainers

### DISCUSSION THREAD

**Discussion by SteffeyDev** [danielbuechele/atemOSC#277] [Announcements]
I haven't used this software myself for several years, and no longer have time to keep maintaining it and carrying it into the future. I know that atemOSC has helped a lot of people, and so I'd love to find someone who is both a developer and a user of this software to take it over. I'd be happy to meet with anyone who is interested to walk them through the code and help with the transition. Reach out if you are interested!

**Comment by randallpacker** [comment]
Hi Peter, you have done amazing work over the years on atemOSC and I wanted to express my appreciation. I use the software daily and it is an integral part of my workflow. If I were a programmer myself I would definitely take it on! 

But I wanted to know what you think of the idea of having atemOSC supported by a company that specializes in media interactivity. Several years ago I spoke with the head of Cycling '74, David Zicarelli about developing a set of Max objects for the ATEM switchers, but then I discovered atemOSC and let it go. But perhaps he might be interested since so much of the work has been done. I use Max and atemOSC to control my switchers and hyperdecks and it would be incredible if it were integrated. I just wanted to know if I could mention it to him again. 

All the best, 

Randall

**Reply by SteffeyDev** [reply]
Thank you for your kind words 😀. I'm open to explore all options, feel free to discuss with him and we can see if that goes anywhere.
